core/src/main/java/com/scalar/db/common/ActiveTransactionManagedDistributedTransactionManager.java

```diff
@@ -42,7 +42,7 @@ public ActiveTransactionManagedDistributedTransactionManager(DatabaseConfig conf
               try {
                 t.rollback();
               } catch (Exception e) {
-                logger.warn("rollback failed", e);
+                logger.warn("rollback failed. transactionId: {}", t.getId(), e);
```

Comment:
Since all exception extends `TransactionException`, would it be possible to append the `transactionId:<txId>`  to the message in the `TransactionException` constructor instead?